docs/developer/respond-to-issue-creation.md

```diff
@@ -0,0 +1,24 @@
+The following example notifies any user who raises an issue with
+insufficient detail in the body, via a direct message in Slack, and
+provides them with a helpful link to the issue. Note that we make use
+of the person available via the `openedBy` field:
+
+```typescript
+export async function requestDescription(inv: NewIssueInvocation) {
+    if (!inv.issue.body || inv.issue.body.length < 10) {
+        await inv.context.messageClient.addressUsers(
+            `Please add a description for new issue ${inv.issue.number}: _${inv.issue.title}_: ${inv.id.url}/issues/${inv.issue.number}`,
+            inv.issue.openedBy.person.chatId.screenName);
```

Comment:
not necessary for this PR, but this is not null safe. We don't have a chatId for every person

Reply:
I'll update it just for the sake of not forgetting it later.